playground/pages/HomeShort.vue

```diff
@@ -1,6 +1,10 @@
 <script setup lang="ts">
 import { useStoryblok } from "@storyblok/vue";
-const story = await useStoryblok("vue", { version: "draft" });
+const story = await useStoryblok(
+  "vue",
+  { version: "draft", resolve_relations: ["Article.author"] }
+  // { resolveRelations: "Article.categories" }
```

Comment:
What would be the idea of this comment? Mainly asking because it has a different format (resolve_relations vs. resolveRelations) and a different value than the one that is being actually used.

The idea is to show that the value will be copied with the change introduced in this PR?

Reply:
The comment is to uncomment to see that the bridge options are overridden, to test it was working properly.